featureflags/evaluations/auth_target.py

```diff
@@ -16,7 +16,7 @@ class Target():
     identifier: str
     name: Union[Unset, str] = UNSET
     anonymous: Union[Unset, bool] = UNSET
-    attributes: Union[Unset, Dict[str, Any]] = UNSET
+    attributes: Union[Unset, Dict[str, str]] = UNSET
```

Comment:
Just for static analysis which will result in a warning. Code will still run if a different type is provided, hence the runtime conversion in analytics.py/165